pygmt/tests/test_binstats.py

```diff
@@ -67,7 +69,13 @@ def test_binstats_quantile():
     assert temp_grid.dims == ("y", "x")
     assert temp_grid.gmt.gtype is GridType.CARTESIAN
     assert temp_grid.gmt.registration is GridRegistration.GRIDLINE
+    assert temp_grid.dtype == "float32"
     npt.assert_allclose(temp_grid.max(), 15047685)
-    npt.assert_allclose(temp_grid.min(), 53)
-    npt.assert_allclose(temp_grid.median(), 543664.5)
-    npt.assert_allclose(temp_grid.mean(), 1661363.6)
+    if Version(__gmt_version__) > Version("6.4.0"):
+        npt.assert_allclose(temp_grid.min(), 53)
+        npt.assert_allclose(temp_grid.median(), 543664.5)
+        npt.assert_allclose(temp_grid.mean(), 1661363.6)
+    else:  # TODO(GMT>=6.5.0): Remove if-condition with different min/median/mean values
+        npt.assert_allclose(temp_grid.min(), 0)
+        npt.assert_allclose(temp_grid.median(), 330700.0)
+        npt.assert_allclose(temp_grid.mean(), 1459889.1)
```

Comment:
Unsure which values are the correct ones. Do we want to check in upstream GMT on what might have changed  `binstat` from GMT 6.4 to 6.5?